OpenSim/Common/ComponentConnector.h

```diff
@@ -857,7 +859,14 @@ class Input : public AbstractInput {
                 "Path to a Component to satisfy the Connector '"            \
                 #cname "' of type " #T " (description: " comment ").")      \
     };                                                                      \
-    /** @endcond                                                         */
+    /** @endcond                                                         */ \
```

Comment:
Update auto generated comment for the property to : "Path to a Component that satisfies the Connector ` ... "